security/cmd/istio_ca/main.go

```diff
@@ -245,6 +248,7 @@ func init() {
 		false, "Enable dual-use mode. Generates certificates with a CommonName identical to the SAN.")
 
 	flags.BoolVar(&opts.sdsEnabled, "sds-enabled", false, "Whether SDS is enabled.")
+	flags.BoolVar(&opts.legacyJwtAllowed, "legacy-jwt-allowed", false, "Whether legacy JWTs are allowed.")
```

Comment:
How many different names for this can we use ? Please make sure exactly same term is used in the proxy setting, values.yaml, flag - and everything else. 

And once again - flags are extremely hard to kustomize, so anything you add as a flag will probably not work with 
the simple install via kubectl .

Reply:
So there are 2 points you that you mentioned here. 
1. As for the naming, could you elaborate it a little more? This is the first time this term is introduced and it is used only in Citadel.

2. This flag is a convenient way for the users to override the default behavior of Citadel. The users can easily change this using helm template. Do you suggest we use something else?